lib/url_pattern.dart

```diff
@@ -1,299 +0,0 @@
-// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
```

Comment:
If `UrlPattern` isn't being used anymore, you should get rid of the last couple of references to it in `client.dart`